doc/source/building/index.rst

```diff
@@ -220,6 +220,60 @@ your system.
         try again. The Fortran compiler should be installed as described in
         this section.
 
+  .. tab-item:: Windows on ARM64
+    :sync: Windows on ARM64
+
+    In Windows on ARM64, the set of a compiler options that are available for
+    building NumPy are limited. Compilers such as GCC and GFortran are not yet
+    supported for Windows on ARM64. Currently, the NumPy build for Windows on ARM64
+    is supported with MSVC toolchain only. The use of a Fortran compiler is more
```

Comment:
I think this limitation is fixed now that #28234 is merged - would you mind pushing a quick edit of this text?